test/test_eos.py

```diff
@@ -89,8 +99,8 @@ def test_pyrometheus_mechanisms(ctx_factory, mechname, y0):
     nodes = thaw(actx, discr.nodes())
 
     # Hand-port chemistry works OK
-    # prometheus_mechanism = UIUCMechanism(actx.np)
-    sol = cantera.Solution(f"{mechname}.cti", "gas")
+    mech_path = str(mechdata / f"{mechname}.cti")
```

Comment:
This won't work if the import location is actually a zip file. Either copy to a temp file and give its name to cantera, or see if cantera will take a file-like. Similar below.

Reply:
I do believe you, though would like to say I was following the [manual](https://importlib-resources.readthedocs.io/en/latest/using.html), down where it goes:
```
In general you never have to worry whether your package is on the file system or in a zip file, as the importlib_resources APIs hide those details from you. Sometimes though, you need a path to an actual file on the file system (...)
To support this, importlib_resources provides an API that will extract the resource from a zip file to a temporary file, and return the file system path to this temporary file as a pathlib.Path object.
```

Reply:
Oh TIL. Nvm in that case.

Reply:
Not so fast, actually.

Reply:
I think that there is something I am doing wrong with that path-to-file approach. This change (6fa2ca3, 37ee711) might actually be better in that it (I think) addresses your original concern, and it has the added value that it actually works.